amr-wind/projection/incflo_apply_nodal_projection.cpp

```diff
@@ -311,9 +329,23 @@ void incflo::ApplyProjection(
         if (!proj_for_small_dt and !incremental) {
```

Comment:
I wanted to confirm that this if condition is correct. (It was there before my changes.)

Reply:
yes -- basically this is never false except during initial iterations

Reply:
I guess I'm not completely sure if it's necessary for these parts, but it should be inconsequential.

Reply:
okay I won't touch it